app/time_tracking/work_time/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.utils.timesince import timesince
+import time
+
+class WorkTime(models.Model):
```

Comment:
any model must have `created` and `updated` fields to understand when the changes were made

Reply:
I added `created_at` and `updated_at` fields to all the models,
I also thought of adding `updated_by`, but because most of our modes are strictly linked to the create user, I didn't add it.